src/backend/routers/paras.test.ts

```diff
@@ -18,3 +20,27 @@ test("getParaById", async (t) => {
   const para = await trpc.getParaById.query({ user_id });
   t.is(para.user_id, user_id);
 });
+
+test("createPara", async (t) => {
+  const { trpc, db, nodemailerMock } = await getTestServer(t, {
+    authenticateAs: "para",
+  });
+
+  await trpc.createPara.mutate({
+    first_name: "Foo",
+    last_name: "Bar",
+    email: "foo.bar@email.com",
+    role: "para",
+  });
+
+  t.truthy(
+    await db
+      .selectFrom("user")
+      .where("first_name", "=", "Foo")
+      .selectAll()
+      .executeTakeFirst()
+  );
+
+  nodemailerMock.mock.getSentMail();
+  console.log(nodemailerMock.mock.getSentMail());
```

Comment:
I would add an assertion something like
```suggestion
  t.true(nodemailerMock.mock.getSentMail().some(mail => mail.subject.includes("confirmation")))
```